src/components/Forms/Onboarding/index.js

```diff
@@ -98,6 +100,11 @@ export default class Form extends React.Component<
     this.setState({ showAdvanced: true });
   };
 
+  handleFinish = () => {
+    localStorage.setItem('chainName', this.state.name);
+    this.setState({ done: true });
```

Comment:
don't know if I'd need a callback here since I'm setting an item in local storage, then setState() is triggered asynchronously so I'm not trying to do anything after setState(), just something before setState()

Reply:
Yup, should have instead mentioned that if we’re gonna the boolean `done`, in let’s say the next line, it might not work as expected, because as you mentioned it too `setState` will be triggered when React finds the right time to do so. I guess what I meant was to just be careful when using `setState`.